ui/app/scripts/directives.js

```diff
@@ -393,5 +393,40 @@ define(['angular', 'xregexp', 'moment'], function(angular) {
           });
         }
       };
+    })
+    .directive('tableSort', function() {
+      function applySort(scope) {
+        if( scope.sortState.sortProperty.toString() === scope.sortProperty.toString() && scope.sortState.sortOrder === 'DESC' ) {
+          scope.sortState.sortOrder = 'ASC';
+        }
+        else if( scope.sortState.sortProperty.toString() === scope.sortProperty.toString() && scope.sortState.sortOrder === 'ASC' ) {
```

Comment:
same comparison here as well. Can we just check if `scope.sortState.sortOrder` and swap the values between `DESC` and `ASC`?